utils/gen-autoargs.py

```diff
@@ -44,7 +44,7 @@
 type_specifier.extend(["std::string"])
 
 # The contents of libnames will be made to genearte library name specifier for arg spec
-libnames = ["", "libstdc++"]
+libnames = ["", "libstdc++", "libc++", "libbfd", "libLLVM"]
```

Comment:
I know this is a bit weird to add such libraries but this commit is added for reference. If library specifier is removed, then we don't need this so we can also make the args/retval list shorter.